### ISSUE TITLE
Tool for viewing and reporting logit diffs between models

### ISSUE THREAD

**Issue by Barahlush** [NelsonG-C/lasr-labs-2025-control-project#60]
We have a setup for RL and ran some trainings, now we need to see if there is any difference in the specific password logits between the trained and untrained models.

I'm developing a tool for that in https://github.com/jonathanbostock/rl-self-conditioning/tree/benchmark
